packages/scoped-elements/package.json

```diff
@@ -35,14 +35,14 @@
     "utils",
     "helpers"
   ],
-  "peerDependencies": {
-    "lit-html": "^1.0.0"
+  "dependencies": {
+    "@open-wc/dedupe-mixin": "^1.1.0",
+    "lit-element": "^2.0.1"
```

Comment:
Not sure if `lit-element` should be added as dependency or peerDependency. :thinking:

Reply:
We depend on lit element, do we not?

Reply:
right 👍